sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerLiveTestBase.cs

```diff
@@ -18,20 +18,45 @@ namespace Azure.Communication.CallingServer.Tests
     public class CallingServerLiveTestBase : RecordedTestBase<CallingServerTestEnvironment>
     {
         // Random Gen Guid
-        protected const string FROM_USER_IDENTIFIER = "e3560385-776f-41d1-bf04-07ef738f2f23";
+        protected const string FROM_USER_IDENTIFIER = "0000000e-2f93-1cfe-69ff-9c3a0d00d763";
 
         // Random Gen Guid
-        protected const string TO_USER_IDENTIFIER = "e3560385-776f-41d1-bf04-07ef738f2fc1";
+        protected const string TO_USER_IDENTIFIER = "0000000e-2f93-61fb-69ff-9c3a0d00d765";
 
         // From ACS Resource "immutableResourceId".
         protected const string RESOURCE_IDENTIFIER = "016a7064-0581-40b9-be73-6dde64d69d72";
 
         // Random Gen Guid
-        protected const string GROUP_IDENTIFIER = "f8c9bb0a-25ec-408d-b335-266dcc0c0c9a";
+        protected const string GROUP_IDENTIFIER = "0000000e-385d-f003-0cf9-9c3a0d00224e";
+
+        // Random Gen Guid
+        protected const string USER_IDENTIFIER = "0000000e-3321-ffd1-69ff-9c3a0d000f47";
+
+        protected const string DOWNLOAD_END_POINT = "https://us-storage.asm.skype.com/v1/objects/Sanitized/content/acsmetadata";
+
+        protected const string DELETE_END_POINT = "https://us-storage.asm.skype.com/v1/objects/Sanitized";
+
+        protected string GetAudioFileUrl()
+        {
+            if (Mode != RecordedTestMode.Playback)
+            {
+                return TestEnvironment.AudioFileUri;
+            }
+            return TestEnvironment.AudioFileUrl;
+        }
+
+        protected string GetDownloadEndPointUrl()
+        {
+            if (Mode != RecordedTestMode.Playback)
```

Comment:
You can pass your placeholder value to .IsSecret() call in the TestEnvironment

```
        public string DownloadEndPoint => GetRecordedVariable(DownloadUrl, options => options.IsSecret(DOWNLOAD_END_POINT));
```

That way you don't have to have your own logic based on the mode.